Create .metadata.json

supernova-at opened the pull request:
Creates the file needed for the repo metadata service.

supernova-at committed:
Create .metadata.json

Creates the file needed for the repo metadata service.

anthoula commented on .github/.metadata.json:
```diff
@@ -0,0 +1,30 @@
+        "enable": false,
+        "frequency": "quarterly",
+        "customName": "UPWARD PHP Static Scan",
+        "branchesToScan": ["develop"]
```
Should `branchesToScan` include "master" ? Looks like `master` is more up-to-date than `develop`

tjwiebell commented:
Please deliver this change to magento-commerce/upward-php. I can check with Mehryar on branch syncing, as @anthoula pointed out, it looks like `develop` is not being synced properly. I think we're also going to delete the `master` branch to align more with core delivery profile, and move away from git-flow.

supernova-at committed:
Adds master branch to scan

Even though the scan is not enabled currently, set it up for future success in case someone wants to turn it on later

supernova-at commented:
Closed in favor of https://github.com/magento-commerce/upward-php/pull/5.
